tests/base_tests.py

```diff
@@ -214,29 +244,6 @@ def test_list_length_error(
         ):
             uninitialized_transformers[self.transformer_name](**args)
 
-    @pytest.mark.parametrize(
```

Comment:
Q: Just wondering, does the TestInit class in 'test_BaseTwoColumnTransformer.py' need updating to inherit 'NewColumnNameInitMixintests' as per 'test_EqualityChecker.py' now this has been removed from here? Asking as TestInit in 'test_BaseTwoColumnTransformer' inherits from 'TwoColumnListInitTests' as per - 

https://github.com/lvgig/tubular/blob/e92f8dc95cb6016a4d730fd1dc6a9977a6b46e69/tests/base/test_BaseTwoColumnTransformer.py#L9

Reply:
yes it does as I moved the NewColumnNameMixin tests into the mixn tests out of TwoColumnListInitTests.  Looks like it does already inherit them to me? Might be missing something here.

Reply:
Ahh okay, this was my first comment but can see from your comments below we are no longer using BaseTwoColumnTransformer